VideoLocker/src/main/java/org/edx/mobile/model/Page.java

```diff
@@ -0,0 +1,45 @@
+package org.edx.mobile.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import java.util.List;
+
+public class Page<T> {
```

Comment:
I have added a commit that centralized the page size definition in `DiscussionAPI`, and removes the individual API method parameters.